edge-modules/iotedge-diagnostics-dotnet/src/GetSocket.cs

```diff
@@ -16,25 +16,41 @@ public static string GetSocketResponse(string server, string endpoint)
             byte[] bytesSent = Encoding.ASCII.GetBytes(request);
             byte[] bytesReceived = new byte[256];
             server = server.Replace("unix://", string.Empty);
+            string page = string.Empty;
 
             // Create a socket connection with the specified server and port.
             using (Socket socket = new Socket(AddressFamily.Unix, SocketType.Stream, ProtocolType.IP))
             {
                 socket.Connect(new UnixDomainSocketEndPoint(server));
```

Comment:
Wait...we're doing `socket.Connect(...)` here, then doing it _again_ inside the `try...catch` a couple lines down...is that intentional?

Reply:
It's wrong. I pushed another commit to fix.